data/102/032/341/102032341-alt-quattroshapes.geojson

```diff
@@ -2,11 +2,30 @@
   "id": 102032341,
   "type": "Feature",
   "properties": {
+    "edtf:cessation":"uuuu",
```

Comment:
Some info about alt geometry properties: https://github.com/whosonfirst/whosonfirst-cookbook/blob/main/how_to/creating_alt_geometries.md#what-rules-must-alt-geometries-follow

WOF hasn't previously added/required the properties being introduced in this PR to alt geometry files, so I'm a little hesitant about folding in these changes. This would impact every WOF repo and increase the scope of work needed for admin updates... the `wof:hierarchy` property alone would be a big change. I'm also not sure if some of these properties are useful in alt files (superseding props, hierarchy label, edtf dates...)

That being said, if there is a need or want to add these properties to all alt files, we should discuss how to best maintain these properties going forward. I don't want to ditch these changes if they're useful to data users.. I just want to be realistic about maintaining any new alt properties being introduced.

Reply:
Yeah I tend to agree, they were introduced by the `exportify` tool, so maybe we can adjust that to not add these fields for alt geoms?

Reply:
Aaron mentioned a new `exportify` tool, so I'll try that and see if it improves things.